README.md

```diff
@@ -69,139 +76,149 @@ Simplified documentation on how to use the tools can be found [here](https://git
 Following are the instructions for running the backups and a restore.
 
 ## Storage
-*Before we get too far into the the details, we're going to take a moment to discuss the most important part of the whole process - **The Storage**.*  The backup container uses two volumes, one for storing the backups and the other for restore/verification testing.  The deployment template separates them intentionally.
 
-The following sections on storage discuss the recommendations and limitations of the storage classes created specifically for the BC Government's [PathFinder](https://console.pathfinder.gov.bc.ca:8443/) environment.
+_Before we get too far into the the details, we're going to take a moment to discuss the most important part of the whole process - **The Storage**._ The backup container uses two volumes, one for storing the backups and the other for restore/verification testing. The deployment template separates them intentionally.
 
+The following sections on storage discuss the recommendations and limitations of the storage classes created specifically for the BC Government's [PathFinder](https://console.pathfinder.gov.bc.ca:8443/) environment.
 
 ### Backup Storage Volume
-The recommended storage class for the backup volume is `nfs-backup`.  This class of storage **cannot** be auto-provisioned through the use of a deployment template.  The `PersistentVolumeClaim` declared in the supplied deployment template for the *backup volume* will purposely fail to properly provision and wire an `nfs-backup` volume if published before you manually provision your `nfs-backup` claim.
+
+NOTE: OCP4 has a different storage class available. TODO
```

Comment:
@garywong-bc, Looks like you meant to update this section with the info on `netapp-file-backup`.

Reply:
frick, it never fails to amaze me how I can muff even the simplest of code changes (i.e. documentation). I had also missed a *.dc.json reference to `nfs-backup` - it's gone now.  TOA